src/containers/CreateDoc/createDocSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
Does this needs to be here ? I mean we should create a reducers folder, so we know where to find them, and also call the file `someReducer.ts` . What do you think ? should we keep it this way ?